src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/SecurityTokenRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Connection.Contracts
+{
+    class RequestSecurityTokenParams 
+    {
+        /// <summary>
+        /// Gets or sets authority.
+        /// </summary>
+        public string Authority { get; set; }
+
+        /// <summary>
+        /// Gets or sets resource.
+        /// </summary>
+        public string Resource { get; set; }
+
+        /// <summary>
+        /// Gets or sets scope.
+        /// </summary>
+        public string Scope { get; set; }
+    }
+
+    class RequestSecurityTokenResponse
+    {
+        /// <summary>
+        /// Gets or sets userName.
```

Comment:
And this should definitely be at least fixed to be the correct name (but the above comment applies here too)

Reply:
Fixed.